Package.swift

```diff
@@ -20,18 +20,17 @@ let package = Package(
     dependencies: [
         // Dependencies declare other packages that this package depends on.
         // .package(url: /* package url */, from: "1.0.0"),
-        .package(
-            name: "Sovran",
-            url: "https://github.com/segmentio/Sovran-Swift.git",
-            from: "1.1.0"
-        )
+        .package(url: "https://github.com/segmentio/Sovran-Swift.git", from: "1.1.0")
```

Comment:
Not related to what you're doing here, but should we bump this to `1.1.1` for parity with [Segment's SDK](https://github.com/segmentio/analytics-swift/blob/main/Package.swift#L24)?

Reply:
Yep. Im planning upgrading several things to be in parity, but in a separate PR.